islandora.info.yml

```diff
@@ -5,30 +5,31 @@ description: "Islandora Core"
 type: module
 package: Islandora
 core: 8.x
```

Comment:
I believe we need to remove this (per the [drupal docs](https://www.drupal.org/docs/creating-custom-modules/let-drupal-know-about-your-module-with-an-infoyml-file#s-specifying-the-core-version-requirement-when-needed) as Islandora relies on functionality introduced in 8.7.x (for example as part of this PR, the [deprecation of `$context` and addition of `$context_definitions`](https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Condition%21Annotation%21Condition.php/property/Condition%3A%3Acontext/8.7.x))

Reply:
@qadan ah i think you're right about context. i don't see anywhere that states that we only support 8.7 and up so before i just want to check with @dannylamb before removing support for that (by removing the `core: 8.x` part)